### ISSUE TITLE
do we want the very last "word" to be the "answer"?

### ISSUE THREAD

**Issue by jovo** [openconnectome/ndprojects#1]

**Comment by j6k4m8** [comment]
Makes sense for numbers "The number of mitos is #." but it gets fuzzy for other stuff ("512 ± 0.5") or non-trivially described values. I left it general (anything in the last output box) so that we can optionally provide one-word answers, OR more, if need be.

**Comment by jovo** [comment]
I see. "Final output" wasn't clear to me. Maybe rephrase, final markdown
thingy?

On Wednesday, January 6, 2016, Jordan Matelsky notifications@github.com
wrote:

> Makes sense for numbers "The number of mitos is #." but it gets fuzzy for
> other stuff ("512 ± 0.5") or non-trivially described values. I left it
> general (anything in the last output box) so that we can optionally provide
> one-word answers, OR more, if need be.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/openconnectome/ndprojects/issues/1#issuecomment-169438833
> .

## 

the glass is all full:  half water, half air.
neurodata.io
